test/integration/testHTAP.js

```diff
@@ -7,50 +7,63 @@ const async = require('async');
 const connOption = require('./connectionOptions').valid;
 const testUtil = require('./testUtil');
 
+function getRandomDBNames() {
```

Comment:
What do you think about creating common method in testUtil.js - with functionality of generating and concatenate suffix with original db name?

Reply:
It seems like Dominik wants to use a uuid, but it does not work with the db or table Name because of the hyphen(-) in the uuid. If we need a function to create a random number as a suffix with the original name, I will create it.

Reply:
We can add this function next time